indico/migrations/versions/20171117_1400_c820455976ba_add_include_description_to_reminders.py

```diff
@@ -0,0 +1,26 @@
+"""Add include_description to reminders
+
+Revision ID: c820455976ba
+Revises: 1d512a9ebb30
+Create Date: 2017-11-17 14:00:05.025539
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = 'c820455976ba'
+down_revision = '1d512a9ebb30'
```

Comment:
`2af245be72a6`

Reply:
Also rename the file to use the current date in the filename; otherwise it's not in the right order with the revisions added in `master` since you opened the PR (ie before changing anything, rebase to the latest upstream master)

Reply:
done